### ISSUE TITLE
fixtures: wrong filetype in authors-list

### ISSUE THREAD

**Issue by pamfilos** [cernopendata/opendata.cern.ch#1811]
filetype is assigned as ```xrootd```. Should be something else or empty

https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/fixtures/data/records/cms-author-list.json#L19036

https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/fixtures/data/records/opera-author-list.json#L538

https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/fixtures/data/records/cms-author-list-Run2011A.json#L19861

**Comment by tiborsimko** [comment]
We have only two types at the moment: 

- `index` meaning that the file is of a "file index" type that is listing location of other data asset files;
- `xrootd` meaning that the file is of a "data asset" type that can be accessed via xrootd protocol

Note that all the various "small" files -- PDF, CSV, PY, PNG etc -- are of the `xrootd` type by default, since they are living on an XRootD-powered backend with have XRootD-enabled endpoint access.  

In other words, not only ROOT files have xrootd type, but all other non-index files... 

Does this create some confusion regarding the file box display or the previewer?  If so, we can naturally amend... 

P.S. The current practice indeed seems to mix the "type" and the "protocol" a bit... plus we also have "extension" to consider and take profit from without duplicating this information...

**Comment by pamfilos** [comment]
ok @tiborsimko . The only reason I created the issue is because I think it is a bit confusing, plus it would help with some UI issue that might appear in the future. 

But, If we like it the current way, we can have workarounds

**Comment by tiborsimko** [comment]
@pamfilos UI-wise, all "index" files should be displayed in a separate box, and all the other "non-index" files should be treated as the main data assets of the record, and displayed in another box in a paginated way.  Is there a concrete issue that would you need to introduce a work around for?

**Comment by tiborsimko** [comment]
(CC also @ArtemisLav regarding data model documentation)

**Comment by tiborsimko** [comment]
As discussed IRL:

1. Let us remove the `xrootd` type value that is basically describing "protocol" rather than "type". (@tiborsimko)

2. Let us display separate the "File indexes" UI box (when `files.type == 'index'`) and "Files" UI box (otherwise). (@pamfilos)

It is not necessary to single out ROOT files, because some datasets are in the ROOT format, some in the CSV format, some in the IG format,etc. (And this information is already covered by `distribution.formats`.) This is similar to how some software files are in Python, some in C++... where we also don't use `files.type` information to code the nature of the files there.
